src/view/writer.js

```diff
@@ -891,6 +924,25 @@ export default class Writer {
 		return newElement;
 	}
 
+	/**
+	 * For given {@link module:engine/view/attributeelement~AttributeElement attribute element}, returns all other
+	 * attribute elements with the same {@link module:engine/view/attributeelement~AttributeElement#id id} that
+	 * were inserted by `Writer` to the view tree.
+	 *
+	 * @param {module:engine/view/attributeelement~AttributeElement} element
+	 * @returns {Set.<module:engine/view/attributeelement~AttributeElement>|null} Set containing all the attribute elements
+	 * with the same `id` or `null` if given element had no `id`.
+	 */
+	getAllClonedElements( element ) {
```

Comment:
The getter should not be a writer method. It should be a method of an element. You should not require creating `view.change` block to call getter, because this block causes rendering.